config.js

```diff
@@ -1,13 +1,16 @@
+require('dotenv-defaults').config({silent: false});
```

Comment:
I would add a comment here explaining that `.env.defaults` is the base level config